app/src/main/java/com/example/internetshop/presentation/viewModel/CategoriesViewModel.kt

```diff
@@ -19,16 +19,19 @@ class CategoriesViewModel @Inject constructor(
 
     val navEventLiveData = SingleLiveEvent<CategoryEvent>()
     val categoriesLiveData = MutableLiveData<List<CategoryViewState>>()
+    val progressBarLiveData = MutableLiveData<Boolean>()
```

Comment:
Try to not use `liveData` in variable names. It's just redundant. `progress` is enough information

Reply:
fixed